lib/hexo/post.ts

```diff
@@ -517,7 +514,8 @@ class Post {
     }).then(() => {
       data.content = cacheObj.escapeCodeBlocks(data.content);
       // Escape all Nunjucks/Swig tags
-      if (disableNunjucks === false) {
+      const hasTag = /(\{\{.+?\}\})|(\{#.+?#\})|(\{%.+?%\})/s.test(data.content);
```

Comment:
Hoist the regexp.

We forgot to do this last time, should not miss this again.